models/staging/ecommerce/stg_ecommerce__order_payments.sql

```diff
@@ -0,0 +1,17 @@
+
+
+select
+    -- primary key
```

Comment:
it would be useful to create a primary key / surrogate key here!

Reply:
Included with commit [e51990f](https://github.com/mtla-saurabh/ae-capstone-sshahdeo/commit/e51990f08f496ab9c2bfbf3e35ddf29c72b95d6e)